bundle/manifests/mongodb-atlas-kubernetes.clusterserviceversion.yaml

```diff
@@ -84,9 +86,6 @@ metadata:
             "credentialsRef": {
               "name": "my-atlas-key",
               "namespace": "mongodb-atlas-system"
-            },
-            "provider": {
-              "name": "mongodbatlas"
```

Comment:
Do we not need this anymore?

Reply:
That's right. It has been removed from the dbaas data structure.